website_sale_recently_viewed_products/README.rst

```diff
@@ -0,0 +1,50 @@
+========================
```

Comment:
You're right, so we should modify the OCA's template. Would you mind making a PR against maintainer-tools explaining this?